### PR TITLE
feat(localizations): Add support for German (de-DE) passkey-text localization

### PR DESCRIPTION
**Pull request by navid-rji**
## Description
The code changes add support for the German passkey-text localization and translations for some texts that haven't been translated.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/localizations/src/de-DE.ts [hunk 1]
```diff
@@ -397,7 +398,7 @@ export const deDE: LocalizationResource = {
       actionLink: 'Anmelden',
       actionLink__use_email: 'E-mail nutzen',
       actionLink__use_email_username: 'E-mail oder Benutzernamen nutzen',
-      actionLink__use_passkey: undefined,
+      actionLink__use_passkey: 'Stattdessen Passkey verwenden',
```
Comment (LekoArts) [inline packages/localizations/src/de-DE.ts:401]: Let's make it shorter

```suggestion
      actionLink__use_passkey: 'Passkey nutzen',
```

File: packages/localizations/src/de-DE.ts [hunk 2]
```diff
@@ -465,9 +466,9 @@ export const deDE: LocalizationResource = {
       'Die Anmeldung ist aufgrund einer fehlgeschlagenen Bot-Validierung fehlgeschlagen. Bitte aktualisieren Sie die Seite, um es erneut zu versuchen, oder wenden Sie sich an den Support, um weitere Unterstützung zu erhalten.',
     form_code_incorrect: '',
     form_identifier_exists: 'Diese E-Mail-Adresse ist bereits vergeben. Bitte wählen Sie eine Andere.',
-    form_identifier_exists__email_address: undefined,
-    form_identifier_exists__phone_number: undefined,
-    form_identifier_exists__username: undefined,
+    form_identifier_exists__email_address: 'Diese E-Mail-Adresse ist bereits vergeben. Bitte wählen Sie eine Andere.',
+    form_identifier_exists__phone_number: 'Diese Telefonnummer ist bereits vergeben. Bitte wählen Sie eine Andere.',
+    form_identifier_exists__username: 'Dieser Benutzername ist bereits vergeben. Bitte wählen Sie einen Anderen.',
```
Comment (LekoArts) [inline packages/localizations/src/de-DE.ts:471]:
```suggestion
    form_identifier_exists__email_address: 'Diese E-Mail-Adresse ist bereits vergeben. Bitte wählen Sie eine andere.',
    form_identifier_exists__phone_number: 'Diese Telefonnummer ist bereits vergeben. Bitte wählen Sie eine andere.',
    form_identifier_exists__username: 'Dieser Benutzername ist bereits vergeben. Bitte wählen Sie einen anderen.',
```

File: packages/localizations/src/de-DE.ts [hunk 3]
```diff
@@ -480,20 +481,20 @@ export const deDE: LocalizationResource = {
     form_password_length_too_short: '',
     form_password_not_strong_enough: 'Passwort nicht stark genug',
     form_password_pwned:
-      'Das gewählte Passwort wurde durch eine Datenpanne im Internet offengelegt. Wählen Sie aus Sicherheitsgründen bitte ein anderes Passwort',
-    form_password_pwned__sign_in: undefined,
+      'Das gewählte Passwort wurde bei einem Datenleck im Internet gefunden. Wählen Sie aus Sicherheitsgründen bitte ein anderes Passwort',
+    form_password_pwned__sign_in: 'Dieses Passwort wurde in einem Datenlecks gefunden und kann nicht verwendet werden. Bitte setzen Sie Ihr Passwort zurück.',
```
Comment (LekoArts) [inline packages/localizations/src/de-DE.ts:485]:
```suggestion
    form_password_pwned__sign_in: 'Dieses Passwort wurde in einem Datenleck gefunden und kann nicht verwendet werden. Bitte setzen Sie Ihr Passwort zurück.',
```

File: packages/localizations/src/de-DE.ts [hunk 4]
```diff
@@ -480,20 +481,20 @@ export const deDE: LocalizationResource = {
...
-    passkey_already_exists: undefined,
-    passkey_not_supported: undefined,
-    passkey_pa_not_supported: undefined,
-    passkey_registration_cancelled: undefined,
-    passkey_retrieval_cancelled: undefined,
+    passkey_already_exists: 'Auf diesem Gerät ist bereits ein Passkey registriert.',
+    passkey_not_supported: 'Passkeys werden auf diesem Gerät nicht unterstützt.',
+    passkey_pa_not_supported: 'Die Registrierung erfordert einen Plattformauthentifikator, der vom Gerät nicht unterstützt wird.',
+    passkey_registration_cancelled: 'Die Passkey-Registrierung wurde abgebrochen oder die Zeit wurde überschritten.',
+    passkey_retrieval_cancelled: 'Passkey verification was cancelled or timed out.',
```
Comment (LekoArts) [inline packages/localizations/src/de-DE.ts:497]:
```suggestion
    passkey_registration_cancelled: 'Die Passkey-Registrierung wurde abgebrochen oder die Zeit wurde überschritten.',
    passkey_retrieval_cancelled: 'Die Passkey-Verifikation wurde abgebrochen oder die Zeit wurde überschritten.',
```

**Review by LekoArts — CHANGES_REQUESTED** [review]
Thanks for the PR! Some smaller adjustments are needed and then it's good to go 👍

**Review by LekoArts — APPROVED** [review]

**Review by dimkl — APPROVED** [review]

**Pull request merged**